### ISSUE TITLE
Can't export `Props` into plain typescript file

### ISSUE THREAD

**Issue by mikestopcontinues** [jonasmerlin/astro-seo#92]
I'm simply trying to:

```ts
import type {Props} from 'astro-seo';
```

But I get this error:

```
Module '"astro-seo"' has no exported member 'Props'.
```

I can successfully import into a `.astro` file. Just not a `.ts` file.

**Comment by jonasmerlin** [comment]
@mikestopcontinues Thanks for reporting this. Is should be fixed in v0.8.3. 

You should now be able to do the following: 

```
import { type SEOProps } from "astro-seo"
```

Can you please try this and report if it works for you now?

**Comment by mikestopcontinues** [comment]
Thanks @jonasmerlin, works like a charm!
